app/models/piece.rb

```diff
@@ -0,0 +1,7 @@
+class Piece < ActiveRecord::Base
```

Comment:
Actually, I think I was confused earlier. Rails doesn't automatically give you support for something like `Piece.pawns`, but I feel like there are so many alternatives (`Pawn.all`, `Piece.find_by(type: "pawn")`, `Piece.where(type: "pawn")`) that the scopes are overkill. I could ask for a team vote though.